website/docs/reference/configs-and-properties.md

```diff
@@ -21,6 +21,21 @@ Whereas you can use **configurations** to:
 * Declare whether a resource should persist its descriptions as comments in the database
 * Apply tags and "meta" properties
 
+### Resource, properties, and configs table
```

Comment:
@mirnawong1 How would you feel about splitting this table into a separate PR?

It feels like the rest of the content of this PR is pretty straight-forward and could be reviewed and merged pretty quickly.

This table has a lot more considerations and might take a lot more back-n-forth.

Reply:
Tackling the table separate might also dove-tail nicely with https://github.com/dbt-labs/docs.getdbt.com/issues/4314

Reply:
Choosing how to display the inheritance order might also dovetail with https://github.com/dbt-labs/docs.getdbt.com/issues/4371

Reply:
One thing I was thinking about the table:
- might be nice to replace `->` with `→`

Another thing:
- I'd love to have links to each the individual configs and properties referenced within the table.

Reply:
ah ok sounds good @dbeatty10 , i'll do that!